google/data_source_google_compute_backend_service_test.go

```diff
@@ -0,0 +1,85 @@
+package google
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccGoogleComputeBackendService_basic(t *testing.T) {
```

Comment:
nit: please remove "Google" from test names?  We're trying to do that as of recently.

Reply:
Thanks for the review @ndmckinley !
I replaced the "Google" with "DataSource" in the tests names,
since just removing this string would name-clash with the resource's own tests, I hope that's ok?